coalib/coala_delete_orig.py

```diff
@@ -1,21 +1,17 @@
 import os
-
-from pyprint.ConsolePrinter import ConsolePrinter
+import logging
```

Comment:
I think I had done similar changes earlier. But @sils denied asked to remove it, since it is out of scope for this PR, and this one is already big enough, I think its better not to include it. And I kinda agree with him.